src/config.js

```diff
@@ -66,17 +72,98 @@ export default class Config {
 	 * @param {*} [value=null] The configuration value. Used if a name is passed to nameOrConfigurations.
 	 */
 	set( name, value ) {
```

Comment:
PS. `set()` docs are outdate. The code samples show incorrect access to the props.